mailbox/api/src/main/java/org/apache/james/mailbox/events/EventDeadLetters.java

```diff
@@ -91,4 +91,6 @@ public String toString() {
     Flux<InsertionId> failedIds(Group registeredGroup);
 
     Flux<Group> groupsWithFailedEvents();
+
+    Mono<Boolean> hasEvent();
```

Comment:
Let me propose `containsEvents` which is more specific than `isEmpty`

Reply:
Agreed